src/derivatives/atlas.md

```diff
@@ -65,10 +63,9 @@ In terms of [`extension`](../glossary.md#extension-common_principles), `nii[.gz]
 Please note that the [`<datatype>/` directory](../glossary.md#data_type-common_principles) is RECOMMENDED.
 The [`<datatype>/` directory](../glossary.md#data_type-common_principles) MAY be omitted in the case
 only one data type (such as `anat/`) is stored under the `tpl-<label>` directory.
```

Comment:
This isn't germane to this PR, so let me know if I should move it to the discussion at bids-standard/bids-specification. 

I like the result of this, but I started trying to write some it out in the schema last night and began re-evaluating my initial thoughts. I guess it follows the same sort of logic as a session entity, so sure.

Reply:
@bendhouseart perhaps you are looking at the wrong place? This is an alternative to my alternative to the BEP. In my alternative, the schema is updated with the specs (or should be).

https://github.com/bids-standard/bids-specification/pull/1856